pom.xml

```diff
@@ -46,53 +77,53 @@
       <version>${jakartaee-api.version}</version>
       <scope>provided</scope>
     </dependency>
-    <!-- Required by Java SE 9+ -->
-    <dependency>
-      <groupId>org.glassfish.jaxb</groupId>
-      <artifactId>jaxb-runtime</artifactId>
-      <version>2.3.8</version>
-      <scope>provided</scope>
-    </dependency>
     <!-- The third-party dependencies this application has -->
     <dependency>
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-lang3</artifactId>
+      <scope>compile</scope>
     </dependency>
     <dependency>
       <groupId>org.primefaces</groupId>
       <artifactId>primefaces</artifactId>
-      <version>12.0.0</version>
+      <version>${primefaces.version}</version>
+      <classifier>jakarta</classifier>
+    </dependency>
+    <dependency>
+      <groupId>org.glassfish.jersey.core</groupId>
+      <artifactId>jersey-server</artifactId>
+      <version>${jersey.version}</version>
+      <scope>provided</scope>
     </dependency>
     <!-- Test dependencies -->
     <dependency>
-      <groupId>junit</groupId>
```

Comment:
These were moved around for no reason. Please put back how it was. Parameterizing the version is fine.